docs/approvals/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Approvals
+description: Approvals, approvals, approvals
+position: 800
+---
+
+
+Octopus currently supports blocking deployments for a project and environment based on the approval status of a Change Request in a connected ServiceNow instance.
```

Comment:
I presume the page is called Approvals (rather than ServiceNow) as we intend to add more ITSM tools in the future possibly? If so, this could be worded more generically to say something like:

>  Octopus currently supports blocking deployments for a project and environment based on the approval status of a Change Request in a supported ITSM tool. Octopus currently supports ServiceNow...

or something :smile:

Reply:
yep, well spotted.